src/matcha_ml/cli/analytics.py

```diff
@@ -0,0 +1,26 @@
+"""Matcha CLI for opting out/in of collecting analytics."""
+import typer
+from rich import print
+
+from matcha_ml.services.global_parameters_service import GlobalParameters
+
+# create a typer app to group all analytics subcommands
+app = typer.Typer()
+
+
+@app.command()
+def opt_out() -> None:
+    """Disable the collection of anonymous usage data."""
+    print(
+        "Data collection has been turned off and no data will be collected - you can turn this back on by running the command: matcha analytics opt-in"
```

Comment:
Nit: I think if you add single quotes around `matcha analytics opt-in`, it'll render it with colours, etc

Reply:
But won't user be confused if to use the command is with/without quotes?